tensorflow/core/lib/io/cache.cc

```diff
@@ -0,0 +1,439 @@
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
```

Comment:
this should be using the TensorFlow copyright

Reply:
Changed.